src/main/java/frc/robot/Robot.java

```diff
@@ -101,10 +113,8 @@ public void autonomousPeriodic() {
 
   @Override
   public void teleopInit() {
-    this.teleopCommand = new TeleopCommand(controls, drive);
-    if (teleopCommand != null) {
-      teleopCommand.schedule();
-    }
+    gyro.resetGyro();
+   // drive.setDefaultCommand(new TeleopCommand(controls,drive, gyro));
```

Comment:
How does the robot drive if there's no teleop command?

Reply:
it glitches occassionally and spins and eventually straightens out again, the encoder drive straight is more reliable. Which means when we merge it I'll save that old code

Reply:
I don't think this is how merging will work. If you want the old encoder code you will need to make sure you're not touching that code. Since this branch is ahead of master, the changes in here will take president